packages/tracing-internal/src/browser/browsertracing.ts

```diff
@@ -222,11 +225,13 @@ export class BrowserTracing implements Integration {
       markBackgroundTransactions,
       traceFetch,
       traceXHR,
-      tracePropagationTargets,
       shouldCreateSpanForRequest,
       _experiments,
     } = this.options;
 
+    const tracePropagationTargets =
```

Comment:
Hmm, does it make sense that the top-level options take _precedence_? 🤔 I think I would expect the options that are passed directly to the integration to "win", so:

```js
this.options.tracePropagationTargets || (clientOptions && clientOptions.tracePropagationTargets);
```

? Not a big thing, as I don't expect this to happen too often, though :) So feel free to ignore.

Reply:
Yeah that's a fair perspective actually. Let's discuss this at our meeting.

Reply:
Based on our meeting we've made two decisions.

1. We'll be combining the arrays of both top level and integration set `tracePropagationTargets`
2. We'll be emitting a warning when both values are set.

Reply:
Detailed below I decided against the combining array approach, but added the warning in b098188